### ISSUE TITLE
features request.

### ISSUE THREAD

**Issue by Harshdeepsinghh** [fakeyatogod/AnimeScrap#15]
episode list redesign- it's really difficult to scroll in anime like one piece. 
volume and brightness control in player

otherwise the app is amazing.

**Comment by fakeyatogod** [comment]
Yeah I was thinking about it before. Couldn't think of any good design so got lazy and forgot.

**Comment by fakeyatogod** [comment]
Need to design episode selector for the better.

**Comment by ThePangel** [comment]
@fakeyatogod idk how or with what you made the UI, if I could help hit me up cause the episode selector is my main drawback from this app

**Comment by fakeyatogod** [comment]
I was thinking of a bottom sheet with a list of episodes and a search view above the list also a ascending and descending toggle

**Comment by ThePangel** [comment]
Seems good, what program sude O use to be compatible, I you want me to help.

**Comment by fakeyatogod** [comment]
Speak English

**Comment by ThePangel** [comment]
Sorry had the auto correct on Spanish since it's my native language, I was saying that if you want some help with the UI what program should I use to design it. That would compatible with your code.

**Comment by fakeyatogod** [comment]
I have zero idea how UI design works.

**Comment by ThePangel** [comment]
You're not the UI designer for this project?

**Comment by fakeyatogod** [comment]
I have just used material components. Did whatever felt right. I dunno how to translate design from figma or Adobe in Android 😓

**Comment by ThePangel** [comment]
To kotlin, I'll look it up.

**Comment by fakeyatogod** [comment]
![Screenshot_2022-10-09-20-45-18-20_d8835f8d092f2f6209ea137bb478b47a](https://user-images.githubusercontent.com/54427421/194764943-f90de93c-4b73-4311-8b3f-9c1c91d4f476.jpg)

Initial UI

**Comment by ThePangel** [comment]
Much better

**Comment by fakeyatogod** [comment]
![Screenshot_2022-10-09-21-48-52-08_d8835f8d092f2f6209ea137bb478b47a](https://user-images.githubusercontent.com/54427421/194768444-1059b8f7-f4b2-450d-a7cb-540defff25cf.jpg)
This should be the final layout

**Comment by ThePangel** [comment]
Great!

**Comment by fakeyatogod** [comment]
In 2.2.4
